theano/sandbox/rng_mrg.py

```diff
@@ -1312,9 +1312,10 @@ def binomial(self, size=None, n=1, p=0.5, ndim=None, dtype='int64',
     def multinomial(self, size=None, n=1, pvals=None, ndim=None, dtype='int64',
                     nstreams=None):
         """
-        Sample `n` (currently `n` needs to be 1) times from a multinomial
+        Sample `n` (currently `n` needs to be > 1) times from a multinomial
         distribution defined by probabilities pvals.
 
+        TODO: MODIFY_ME
```

Comment:
If this comment needs a modification because of the changes you did in this PR, you should update this comment yourself.